apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -41,17 +40,13 @@ describe("Client API", () => {
         accessTokenBasic = token;
       },
     });
-    getLocalStorage({
-      storageKey: "jwt-auth-m2m",
-      callback: (token: string) => {
-        accessTokenM2M = token;
-      },
-    });
   };
 
   before(() => {
+    cy.wait(61000);
```

Comment:
This is painful, but anything less than this resulted in substantially less successful runs.

Reply:
Waiting 60 seconds isn't reasonable. Are we being rate limited by accounts, or by ips, or both?

Reply:
See my PR description for more info. I believe we are limited by IP.